LICENSE

```diff
@@ -0,0 +1,171 @@
+Copyright 2020-Present Okta, Inc.
```

Comment:
nit: I don't think you can copyright an existing license ;)

Reply:
I copied it from here: https://github.com/okta/okta-sdk-java/blob/master/LICENSE#L1 :)

Reply:
Updated per your above recommendation which looks better.